clouddq/main.py

```diff
@@ -209,7 +209,7 @@
     help="If True, allows looking up entity_uris with scheme 'bigquery://' "
     "using Dataplex Metadata API. ",
     is_flag=True,
-    default=False,
+    default=True,
```

Comment:
`hidden=True`?

Reply:
this will hide this from help output. Should we do that. Please suggest.

Reply:
With the updated description, we don't need to hide it.